### PR TITLE
More accurate fee calculation in CreateDenominated

### PR DESCRIPTION
**Pull request by UdjinM6**

### CODE REVIEW

File: src/privatesend/privatesend-client.cpp [hunk 1]
```diff
@@ -1504,8 +1505,15 @@ bool CPrivateSendClientSession::CreateDenominated(CAmount nBalanceToDenominate,
     std::vector<CRecipient> vecSend;
     CKeyHolderStorage keyHolderStorageDenom;
 
+    CCoinControl coinControl;
+    // Every input will regure at least this much fees in duffs
```
Comment (xdustinface) [inline src/privatesend/privatesend-client.cpp:1509]: I guess that should mean "require"? 
```suggestion
    // Every input will require at least this much fees in duffs
```

File: src/privatesend/privatesend-client.cpp [hunk 2]
```diff
@@ -1504,8 +1505,15 @@ bool CPrivateSendClientSession::CreateDenominated(CAmount nBalanceToDenominate,
...
+    const CAmount nInputFee = GetMinimumFee(148, coinControl, ::mempool, ::feeEstimator, nullptr /* feeCalc */);
+    // Every output will regure at least this much fees in duffs
```
Comment (xdustinface) [inline src/privatesend/privatesend-client.cpp:1511]: same here
```suggestion
    // Every output will require at least this much fees in duffs
```

**Comment by xdustinface** [comment]
utACK

**Comment by PastaPastaPasta** [comment]
backported in https://github.com/dashpay/dash/pull/3670

**Pull request merged**